<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>WdgImportImageSequence</class>
 <widget class="QWidget" name="WdgImportImageSequence">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="lstFiles">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btnAddImages">
           <property name="text">
            <string>Add images...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRemove">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Order</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QComboBox" name="cmbOrder"/>
          </item>
          <item>
           <widget class="QComboBox" name="cmbSortMode"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autoAddHoldframesBox">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Adds the imported frames in their respective slots as per the sequence numbers in the file name&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Add hold frames automatically</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="beginFromZeroIndexBox">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The first imported frame will be placed in #0 instead of #1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Begin frames from index 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Timing</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Start at</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="KisIntParseSpinBox" name="spinFirstFrame">
            <property name="toolTip">
             <string>The frame number for the first image</string>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Step</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="KisIntParseSpinBox" name="spinStep">
            <property name="toolTip">
             <string>Number of frames between images</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>999</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QLabel" name="lblFramerate">
            <property name="text">
             <string>Source FPS:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisIntParseSpinBox</class>
   <extends>QSpinBox</extends>
   <header>kis_int_parse_spin_box.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
